pkg/skaffold/schema/latest/config.go

```diff
@@ -392,6 +392,10 @@ type KubectlDeploy struct {
 	// RemoteManifests lists Kubernetes manifests in remote clusters.
 	RemoteManifests []string `yaml:"remoteManifests,omitempty"`
 
+	// DisableValidation passes the `--valdiate=false` flag to supported
+	// `kubectl` commands when enabled.
+	DisableValidation bool `yaml:"validate,omitempty"`
+
 	// Flags are additional flags passed to `kubectl`.
 	Flags KubectlFlags `yaml:"flags,omitempty"`
```

Comment:
I wonder if `DisableValidation ` could be under `KubectlFlags ` somehow?

Reply:
I don't think we want it under `KubectlFlags`, because those flags are applied for any `kubectl` command we run, and `--validate` is only a valid option on some commands.

Reply:
I meant, could we add a `DisableValidation` field to `KubectlFlags` and apply that to supported `kubectl` commands.

Reply:
Oh, right.  That makes a lot of sense!